app/main.py

```diff
@@ -1,3 +1,18 @@
-def cache(func):
-    # Write your code here
-    pass
+import time
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    cache_result = {}
+
+    def add_result(*args, **kwargs) -> Callable:
+        if args in cache_result and cache_result[args][0] >= time.time():
```

Comment:
Why do you need time.time() in this solution?

Reply:
I see, I don't need that, I don't remember what I thought